src/DelegateInterface.php

```diff
@@ -8,7 +8,7 @@
 interface DelegateInterface
 {
     /**
-     * Dispatch the next available middleware and return the response.
+     * Process an incoming server request and return the response.
```

Comment:
The documentation for the PSR already uses the term "dispatch", so switching this to "process" doesn't make sense.